ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml

```diff
@@ -201,6 +201,44 @@ tests:
     test:
     - chain: openshift-qe-node-density-cni
     workflow: openshift-qe-installer-aws
+- as: control-plane-ipsec-120nodes
+  cron: 0 3 22 * *
```

Comment:
In this [doc](https://docs.google.com/document/d/1NbXwfZwSkMdftuKW1Ob705rQk-XHLtPtP1d1vuqF9Vo/edit#heading=h.an1n9la7teu5) Cron Rules for 'control-plane-120nodes' monthly is either 4, 18, 11, or 25. 
CC: @afcollins

Reply:
Updated, will be pushing new PR